dashboard/lib/pd/foorm/rollup_helper.rb

```diff
@@ -36,12 +36,13 @@ def self.get_rollup_question_data(parsed_forms, question_details)
         # for now we will skip questions that don't have the same type and choices/rows/columns
         # across all versions/forms.
         next unless validate_question(question, question_data, parsed_forms)
-        parsed_question_data = parsed_forms[question_data[:form_keys].first][question]
+        parsed_question_data = parsed_forms[question_data[:form_type]][question_data[:form_keys].first][question]
```

Comment:
Epic.  Should this line be broken up in to a few steps? :)

Reply:
updating in a few other places too, good point :)